include/rocksdb/manifest_ops.h

```diff
@@ -0,0 +1,30 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
```

Comment:
FYI for new files we have "Meta Platforms" copyright text (can find it with `git grep`)